src/JasperPHP/JasperPHP.php

```diff
@@ -95,7 +105,8 @@ public function process($input_file, $output_file = false, $format = array("pdf"
             $command .= " -P";
             foreach ($parameters as $key => $value)
             {
-                $command .= " " . $key . "=" . $value;
+              if(!$filter_params || in_array($key, $params_output))
```

Comment:
Hi @cossou!

It must add parameter only when is not filtering parameters or when parameter exists in output.
I've created this filter because my app have forms who process multiple reports who have different parameters.